src/components/countdown/CountdownSettings.vue

```diff
@@ -17,10 +23,29 @@ import BaseSettings from '../presentation/BaseSettings.vue'
 
 export default {
   extends: BaseSettings,
+  data () {
+    return {
+      klokopties: [
```

Comment:
De codebase is in het Engels geschreven (alles behalve user-facing strings). Kun je jouw NL variabelen vertalen naar het Engels? Misschien kun je hiervan `types` maken? Aangezien het de verschillende types zijn die je voor de `Countdown` kunt kiezen

Reply:
Is goed ga ik proberen aan te denken. Werk zelf normaal in nl ivm mijn kennis van en.